extra/cache-extra/TokenParser/CacheTokenParser.php

```diff
@@ -31,24 +31,22 @@ public function parse(Token $token): Node
         $tags = null;
         while ($stream->test(Token::NAME_TYPE)) {
             $k = $stream->getCurrent()->getValue();
+            if (!in_array($k, ['ttl', 'tags'])) {
+                throw new SyntaxError(\sprintf('Unknown "%s" configuration.', $k), $stream->getCurrent()->getLine(), $stream->getSourceContext());
+            }
+
             $stream->next();
-            $args = $expressionParser->parseNamedArguments();
+            $stream->expect(Token::PUNCTUATION_TYPE, '(');
+            if ($stream->test(Token::PUNCTUATION_TYPE, ')')) {
+                throw new SyntaxError(\sprintf('The "%s" modifier takes exactly one argument (0 given).', $k), $stream->getCurrent()->getLine(), $stream->getSourceContext());
```

Comment:
shouldn't we keep this a similar better error message after the arg, when passing multiple args, instead of saying `Unexpected token "punctuation" of value "," ("punctuation" expected with value ")"` ?

Reply:
Sure, done